### ISSUE TITLE
Develop manual testing strategy for mcnp_import

### ISSUE THREAD

**Issue by gonuke** [svalinn/DAGMC-Trelis#25]
This is a long-standing oft-discussed issue.  We should capture ideas somewhere and link here.

**Comment by pshriwise** [comment]
There's been a lot of repeating discussion about this issue. I'll summarize here so we avoid having this discussion again in the future:

The pitfall we keep falling into when considering testing of the import tool is the testing that the CAD engine is creating the correct volumes from the mcnp2cad geometry by somehow checking the resulting CAD model. 

The real problem at hand is testing the mcnp2cad logic and its calls into the CAD engine. To do this some kind of mock iGeom/DAGMC convenience interface has repeatedly been discussed. 

Another optional issue is to test that calls into this CAD interface are doing what we expect, these could be pulled in from the iGeom interface in CGM.

TL;DR

 Need two layers of testing to be fully robust:

 - One layer testing that the calls to the CAD engine are correct using mock interface
 - Another optional layer testing that the iGeom calls are creating the correct CAD objects.

**Comment by pshriwise** [comment]
We didn't specify this, but I'd imagine these would be built/executed the same way the export tests #24 will be setup.
